config/deploy/docker/lib/cron_installer.rb

```diff
@@ -8,15 +8,19 @@
 # Run from rails root
 
 class CronInstaller
+  attr_accessor :cluster
   MAX_DESCRIPTION_LENGTH = 512
 
   def run!
+    self.cluster = ENV.fetch('AWS_CLUSTER') { ENV.fetch('AWS_PROFILE') { ENV.fetch('AWS_VAULT') } }
```

Comment:
@yakloinsteak any idea what I need to get these ENV?  They seem to be available in RollOut.

Reply:
Just set `AWS_CLUSTER=openpath` in your environment. I use direnv, and when one uses aws-vault, that one gets set, so it assumes the vault name matches the cluster name. We should just have an explicit cluster variable like AWS_CLUSTER. I think we kept AWS_PROFILE since we both used that at one point. I can look with you if this just muddied the waters more.